### PR TITLE
merge lums changes

### PR DESCRIPTION
**Pull request by zubair-arbi**
Merge `Lums` changes in core edly `edx-platfrom` in `Lums` own fork of `edx-platfrom`.

### CODE REVIEW

**Comment by zubair-arbi** [comment]
I have updated the `develop` with the previous changes in the `edly -> edx-platfrom` branch `develop`.

File: openedx/features/lums/models.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    SINDH = 'sindh'
+    KPK = 'kpk'
+    BALOCHISTAN = 'balochistan'
+
+    PROVINCE_CHOICES = (
+        (PUNAJB, _('Punjab')),
+        (SINDH, _('Sindh')),
+        (KPK, _('Khyber Pakhtunkhwa')),
+        (BALOCHISTAN, _('Balochistan')),
+    )
```
Comment (tasawernawaz) [inline openedx/features/lums/models.py:24]: do we want to have `Gilgit-Baltistan`?

File: openedx/features/lums/admin.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+from django.contrib import admin
+
+from openedx.features.lums.models import UserProfileExtension
+
+
+class UserProfileExtensionAdmin(admin.ModelAdmin):
+    """ Admin interface for the UserAttribute model. """
```
Comment (tasawernawaz) [inline openedx/features/lums/admin.py]: I think there is a typo, this is for `UserProfileExtension`

File: openedx/features/lums/README.rst [hunk 3]
```diff
@@ -0,0 +1,5 @@
+Lums
+-----
+
+This directory contains a Django application to add custom functionality for
+lums features within edx-platform.
```
Comment (tasawernawaz) [inline openedx/features/lums/README.rst]: nit lums --> LUMS SeO

**Review by tasawernawaz — COMMENTED** [review]
I believe this code is already reviewed, added a couple of small comments.

File: openedx/features/lums/models.py (hunk 1 above)
Comment (zubair-arbi) [reply openedx/features/lums/models.py:24]: No. The product owner specifically asked to add only 4 provinces.

File: openedx/features/lums/admin.py (hunk 2 above)
Comment (zubair-arbi) [reply openedx/features/lums/admin.py]: Done

File: openedx/features/lums/README.rst (hunk 3 above)
Comment (zubair-arbi) [reply openedx/features/lums/README.rst]: Done

**Comment by zubair-arbi** [comment]
@tasawernawaz Yes the changes in this PR have already been reviewed.
I have also addressed your feedback.

**Comment by zubair-arbi** [comment]
I am going to merge these changes in `develop` branch. All the development will be done in the `develop` branch this way we can always rebase our master branch on any future openedx release without any problem.

**Pull request merged**